src/main/java/book_store/model/User.java

```diff
@@ -0,0 +1,29 @@
+package book_store.model;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Data
+@SQLDelete(sql = "Update users SET is_deleted = TRUE WHERE id = ?")
+@Where(clause = "is_deleted=FALSE")
```

Comment:
```suggestion
@SQLRestriction(clause = "is_deleted = FALSE")
```

Reply:
I can't use the @SQLRestriction(clause = "is_deleted = FALSE") annotation I don't have a "clause" method in it